test/test.js

```diff
@@ -58,14 +58,19 @@ describe('transit', function() {
       d: '2'
     }, 'bar');
 
+    class BazRecord extends Immutable.Record({
```

Comment:
Yes it's a possibility, but the old syntax is unreadable and awful ;)

I can eventually create another file to just ignore it.